infra/terraform/aws/modules/feast/variables.tf

```diff
@@ -0,0 +1,63 @@
+variable "region" {
+}
+
+variable "name_prefix" {
+}
+
+variable "vpc_id" {
+}
+
+variable "subnet_filter_tag" {
+  type = map(string)
+  default = {
+          Tier = "private"
+        }
+}
```

Comment:
what is this param for on the feast module? given that we also have `subnets`

Reply:
will be removed. missed while cleaning up.